examples/python/ToF/tof_rvc4_exposed_settings.py

```
@@ -0,0 +1,450 @@
+        dev = p.getDefaultDevice()
+        if dev.getPlatform() != dai.Platform.RVC4:
+            print(f"ERROR: this example targets RVC4 devices, got {dev.getPlatform()}", file=sys.stderr)
+            return 2
+
+        p.start()
+        # Also exercise the runtime path: re-send the same config after start.
+        inputConfigQueue.send(cfg)
+
+        if args.test:
+            # Count frames per stream in which at least one non-zero pixel was produced.
+            got = {name: 0 for name in outputQueues}
```

Ruff (0.15.21)

[warning] Unnecessary dict comprehension for iterable; use `dict.fromkeys` instead

Replace with `dict.fromkeys(iterable)`)

(C420)